src/modules/esl-media/README.md

```diff
@@ -88,7 +145,7 @@ using a single tag as well as work with external providers using simple native-l
 *(note: that feature doesn't work for Abstract Iframe provider, also doesn't work for HTMLAudio and HTMLVideo providers in case when Web-server when hosted resource doesn't support ['Accept-Ranges' HTTP response marker](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Ranges))*
  - `focusable` (boolean) - marker that allows the video to be focused by keyboard navigation. By default, the video is focusable if `controls` are enabled.
 
-#### Deprecated attributes (going to be removed in the next major release):
+#### Deprecated attributes (going to be removed in 5.0.0):
```

Comment:
`5.1.0` ?

Reply:
They are already )) Doc was not updated 5.0.0 clenup.